packages/chat-component-bindings/src/messageThreadSelector.ts

```diff
@@ -217,3 +217,13 @@ const sanitizedMessageContentType = (type: string): MessageContentType => {
     ? lowerCaseType
     : 'unknown';
 };
+
+const messagesWithContentOrFileSharingMetadata = (message: ChatMessageWithStatus) => {
+  if (message.deletedOn) {
+    return false;
+  }
+  if (message.metadata?.['fileSharingMetadata']) {
+    return true;
+  }
+  return message.content?.message !== '';
+};
```

Comment:
This condition is a bit different from previous one - the message without content will be still shown
We should also hide those messgaes with message.content===undefined

Reply:
Especially in teams interop scenarioes, teams is sending a lot noise message(at least we don't handle them for now) which message.content===undeifined